openwisp_controller/config/tests/test_views.py

```diff
@@ -69,46 +69,74 @@ def test_get_default_templates(self):
             inactive_t,
         ) = self._create_template_test_data()
         self._login()
-        response = self.client.get(
-            reverse('admin:get_default_templates', args=[org1.pk])
-        )
-        templates = response.json()['default_templates']
-        self.assertEqual(len(templates), 2)
-        self.assertIn(str(t1.pk), templates)
-        self.assertIn(str(t3.pk), templates)
-        response = self.client.get(
-            reverse('admin:get_default_templates', args=[org2.pk])
-        )
-        templates = response.json()['default_templates']
-        self.assertEqual(len(templates), 2)
-        self.assertIn(str(t2.pk), templates)
-        self.assertIn(str(t3.pk), templates)
+        with self.assertNumQueries(4):
```

Comment:
2 queries are made for authentication, 1 for getting organization and 1 for getting templates